src/components/Link/Link.js

```diff
@@ -68,13 +70,39 @@ class Link extends Component {
       ...props
     } = this.props;
 
+    let linkItemClassNames;
+
+    // If link is not a relative link, or if link should open in new window,
+    // then directly us an `a` tag, insted of the `NextLink` component
+    if (linkItem) {
+      const { navigationItem: { data } } = linkItem;
```

Comment:
I don't like coupling this component with the `linkItem` data structure. Can you just add `isUrlRelative` and `shouldOpenInNewWindow` props, and use the existing `className` prop? Then set those props in each place where this is used?

If that's too much duplicated logic, then you could make a `NavItemLink` component that wraps this one and have that one only accept `linkItem` prop.